python/ray/_private/runtime_env/packaging.py

```diff
@@ -82,32 +82,100 @@ def _xor_bytes(left: bytes, right: bytes) -> bytes:
 
 def _dir_travel(
     path: Path,
+    root: Path,
     excludes: List[Callable],
-    handler: Callable,
+    includes: List[Callable],
+    handler: Callable = None,
     logger: Optional[logging.Logger] = default_logger,
 ):
-    """Travels the path recursively, calling the handler on each subpath.
-
-    Respects excludes, which will be called to check if this path is skipped.
     """
-    e = _get_gitignore(path)
+    Recursively traverses the directory at path, calling the handler on each subpath.
+    Supports both `includes` and `excludes`:
+    - `excludes`: A list of callables to check if a path should be skipped.
+    - `includes`: A list of callables to check if a path should be processed.
```

Comment:
API wise, if a path is specified in both `includes` and `excludes`, what the behavior should be?